samples/csharp_dotnetcore/52. twilio-adapter/DeploymentTemplates/template-with-new-rg.json

```diff
@@ -1,183 +1,183 @@
-{

```

Comment:
What was the change in this file? If it was the end of line, you shouldn't apply that change

Reply:
Let's leave this for now.